screens/StoreLookupScreen.js

```diff
@@ -50,41 +73,62 @@ export default class StoreLookupScreen extends React.Component {
     this.setState({ store, storePermission });
   };
 
+  handleChangeText = searchStr => {
+    this.setState({
+      searchStr
+    });
+    this.updateFilteredStores(searchStr);
+  };
+
   handleNavigate = () => {
     this.props.navigation.navigate('ClerkLogin', { store: this.state.store, storeName: this.state.store.storeName });
   };
 
+  updateFilteredStores = searchStr => {
+    this.setState({
+      filteredStores: this.state.stores.filter(store => store.storeName.toLowerCase().includes(searchStr.toLowerCase()))
+    });
+  };
+
   render() {
     return (
       // TODO break out this onChange into a function
-      <CheckInContainer>
-        <CheckInContentContainer>
-          <Title color="#fff">Enter store name</Title>
-          <TextField
-            style={{ marginTop: 32 }}
-            placeholder="ex: Healthy Corner Store"
-            onChangeText={text => this.storePermissionHandler(text)}
-            value={this.state.store.storeName}
-          />
-          <FilledButtonContainer
-            style={{ marginTop: 32 }}
-            color={this.state.storePermission ? Colors.primaryGreen : Colors.lightestGreen}
-            width="253px"
-            height="40px"
-            onPress={() => this.handleNavigate()}
-            disabled={!this.state.storePermission}>
-            <ButtonLabel color="white">Next</ButtonLabel>
-          </FilledButtonContainer>
-        </CheckInContentContainer>
-
-        {/* <Picker mode="dropdown" onValueChange={store => this.setState({ store })} selectedValue={this.state.store}>
-          {this.state.stores.map(store => {
-            return <Picker.Item label={store.storeName} value={store} key={store} />;
-          })}
-        </Picker>
-        {this.state.errorMsg ? <Text>{this.state.errorMsg}</Text> : null} */}
-        {/* <Button title="Testing Bypass" onPress={() => this._devBypass()} /> */}
-      </CheckInContainer>
+      <DismissKeyboard>
+        <CheckInContainer>
+          <CheckInContentContainer>
```

Comment:
Slightly annoying to click once to dismiss the keyboard then click again to select the store I want